### DISCUSSION TITLE
Using Wazuh for FIM of logs

### DISCUSSION THREAD

**Discussion by juddcbrown** [Security-Onion-Solutions/securityonion#5122] [Unsupported Versions] [answered]
Hello,

A re-ask of this question: PCI 10.5.5 requires FIM to monitor stored logs. How can Wazuh be used to hash and monitor log files or directories?

Any ideas appreciated. Being hounded by Audit.

**Comment by DinhDiesel** [comment, marked as answer]
"< directories check_all="yes" realtime="yes" report_changes="yes" > C:/apple< /directories >"

this is what I've used to monitor files in ossec.conf

https://documentation.wazuh.com/current/learning-wazuh/detect-fs-changes.html
